cfme/tests/infrastructure/test_tenant_quota.py

```diff
@@ -582,3 +583,134 @@ def test_quota_not_fails_after_vm_reconfigure_disk_remove(request, appliance, fu
+    """This fixture creates new group and assigned by new project"""
+    role = appliance.collections.roles.instantiate(name="EvmRole-user_self_service")
+    user_role = role.copy(
+        name=fauxfactory.gen_alphanumeric(25, "self_service_role_"),
+        vm_restriction="None"
+    )
+    group = appliance.collections.groups.create(
+        description=fauxfactory.gen_alphanumeric(start="group_"),
+        role=user_role.name,
+        tenant=f"My Company/{new_project.parent_tenant.name}/{new_project.name}",
+    )
+    yield group
+    group.delete_if_exists()
+
+
+@pytest.fixture(scope="module")
+def new_user_project(appliance, new_group_project):
+    """This fixture creates new user which is assigned to new group and project"""
+    user = appliance.collections.users.create(
+        name=fauxfactory.gen_alphanumeric(start="user_").lower(),
+        credential=Credential(principal=fauxfactory.gen_alphanumeric(start="uid"),
+                              secret=fauxfactory.gen_alphanumeric(start="pwd")),
+        email=fauxfactory.gen_email(),
+        groups=new_group_project,
+        cost_center="Workload",
+        value_assign="Database",
+    )
+    yield user
+    user.delete_if_exists()
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1456819, 1401251])
+@pytest.mark.parametrize(
+    ['set_project_quota', 'custom_prov_data', 'set_default'],
+    [
+        [(('cpu', '2'), ('storage', '0.01'), ('memory', '2')),
+         ({'hardware': {'num_sockets': '8', 'memory': '4096'}}), False]
+    ],
+    indirect=['set_project_quota', 'custom_prov_data', 'set_default'],
+    ids=['max_cpu_storage_memory']
+)
+@pytest.mark.parametrize('context', [ViaSSUI, ViaUI])
+@pytest.mark.provider([RHEVMProvider], scope='function')
+def test_simultaneous_tenant_quota(request, appliance, context, new_project, new_user_project,
+                                   set_project_quota, custom_prov_data, catalog_item, set_default):
+    """
+    Bugzilla:
+        1456819
+        1401251
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/6h
+        caseimportance: medium
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.8
+        casecomponent: Provisioning
+        tags: quota
+        setup:
+            1. Create tenant1 and tenant2.
+            2. Create a project under tenant1 or tenant2
+            3. Enable quota for cpu, memory or storage etc
+            4. Create a group and add role super_administrator
+            5. Create a user and add it to the group
+        testSteps:
+            1. Login with the newly created user in the service portal. Take multiple items which go
+               over the allocated quota
+        expectedResults:
+            1. CFME should deny request with quota exceeded reason
+    """
+    with new_user_project:
+        with appliance.context.use(context):
+            appliance.server.login(new_user_project)
```

Comment:
Why do you login again with the `new_user_project`? As far as I know, when you do `with new_user_project` it automatically logs in with the `new_user_project` and performs the actions.

Reply:
Yeah, I also tried what you are suggesting but this is the user(not having admin rights). Logs in to SSUI portal and order the catalog item. So this needs explicitly login using ```appliance.server.login()```